router/pump.go

```diff
@@ -405,3 +411,22 @@ func (cp *containerPump) remove(logstream chan *Message) {
 	defer cp.Unlock()
 	delete(cp.logstreams, logstream)
 }
+
+func parseLogLine(line string, originalTimestamps bool) (string, time.Time) {
+	line = strings.TrimSuffix(line, "\n")
+
+	if ! originalTimestamps {
+		return line, time.Now()
+	}
+
+	logEntry := strings.SplitN(line, " ", 2)
+	logTime, err := time.Parse(time.RFC3339Nano, logEntry[0])
+	if err != nil {
+		return line, time.Now()
+	}
+
+	if len(logEntry) == 2 {
+		return logEntry[1], logTime
+	}
+	return "", logTime
```

Comment:
if `line` has no white spaces - if it had the function would finish at `len(logEntry) == 2` - and `logEntry[0]` is a valid time string - otherwise the function would finish at `err != nil` then it means that an empty message was logged